### DISCUSSION TITLE
Compatibility List for Exx50

### DISCUSSION THREAD

**Discussion by SkyrilHD** [SkyrilHD/Dell-Exx50-Hackintosh#2] [Q&A] [answered]
This discussion about the Exx50 models. Since I only own an E7250 which is the only device I can test on, I do not know if the EFI is compatible with other models of Exx50 models. So, it would be useful if you could answer with the model you have and whether it working or not. If the EFI is not working, consider reporting the issue which I would look into.

**Comment by gaslarfa** [comment, marked as answer]
I confirm that this EFI works perfectly on DELL E7450, I have already been using it for two weeks even until I update to BIG SUR 11.2 as if it were a real MAC.
Thank you very much for the EFI, the truth is that I had about a year looking for one that was as compatible as this one.

**Reply by SkyrilHD** [reply]
Glad to hear it works well! Have fun

**Reply by gaslarfa** [reply]
WI-Fi is work on new update...
You are the best bro, thanks you so much!!!

**Reply by mtellect** [reply]
Works too on Latitude E7450. well done....

**Reply by shelterx** [reply]
Wifi card (AC7265) did not work for me on the DELL E7450 for some reason, I know for a fact that the card wasn't faulty. And the OpenIntelWireless devs didn't really have an answer to why it didn't work,  Perhaps I had some weird revision of the AC 7265 card.

But when I changed to an Intel Wireless AC8260 card it started working right away.

**Reply by tharindupeiris0425** [reply]
Works Perfectly in Dell Latitude 3450 
Intel Core i5-5200U
SK Hynix 8GB
addlink 256GB SSD
Intel HD 5500 Graphics

**Reply by FireSpiritGM** [reply]
I have a question, So i have a dell latitude e7450 with an i5 5300u and 16 gb ram and a geforce 840m, is there a way for me to dualboot windows and macOS where i can have windows run no the dedicated GPU and macOS run on the iGPU
would appreciate any responses

**Reply by SkyrilHD** [reply]
@FireSpiritGM The EFI already does that, however I need someone to provide me the ACPI path for the dGPU, so I can disable it on macOS properly. An advantage would be that we will not rely on a boot-arg, but rather disabling on ACPI level. Also, it would help with power consumption in macOS.

One can follow the guide here: https://dortania.github.io/Getting-Started-With-ACPI/Desktops/desktop-disable.html

**Reply by FireSpiritGM** [reply]
i appreciate the fast response!, ill see what i can do and come back if any issues presist

**Reply by FireSpiritGM** [reply]
![image](https://github.com/SkyrilHD/Dell-Exx50-Hackintosh/assets/134444592/621f7c5d-0b77-463f-9baf-86fd59d10c3d)

**Reply by FireSpiritGM** [reply]
i think this is what you are talking about, i aplogize for my lack of knowledge, i just don't really know how to convert it to a path, i have done some modding on the laptop, removed the battery, replaced the screen and upgraded the ram, this is the second ACPI -ACPI(_SB_)#ACPI(PCI0)#ACPI(RP05)#ACPI(D034)

**Reply by SkyrilHD** [reply]
Can you try this EFI?

[Dell-Exx50-Hackintosh-3.2.Monterey-ProperGPUDisable-NoAirportItlwm.zip](https://github.com/SkyrilHD/Dell-Exx50-Hackintosh/files/11652669/Dell-Exx50-Hackintosh-3.2.Monterey-ProperGPUDisable-NoAirportItlwm.zip)

Can you check if the dGPU is still present on Windows, and can you check if the dGPU is now completely removed from macOS?
I had to remove AirportItlwm from the EFI, since GitHub only allows me to upload up to 25 MB.

**Reply by FireSpiritGM** [reply]
im having a problem with the installer, i made the bootable drive using macrecovery and the disk managements method, when i boot, 2 options show and then it black screens and it does that when i try to make a selection,  any help?

**Reply by SkyrilHD** [reply]
The EFI, I just provided, was for desktop GPUs anyway (oops).

Here is the EFI with Bumblebee method: 
[Dell-Exx50-Hackintosh-3.2.Monterey-BumblebeeGPUDisable-NoAirportItlwm.zip](https://github.com/SkyrilHD/Dell-Exx50-Hackintosh/files/11652758/Dell-Exx50-Hackintosh-3.2.Monterey-BumblebeeGPUDisable-NoAirportItlwm.zip)


Also, can you elaborate more, "it black screens"? Do you see the OpenCore boot picker after selecting the USB? Also, double-check your BIOS settings with the README.

**Reply by FireSpiritGM** [reply]
ok, so when i boot, it shows me 2 options, Untitled - Macos (the drive ), and when i try to make a selection it black screens and i have to reboot

**Reply by FireSpiritGM** [reply]
also im trying to install ventura and this is a Monetery efi, is there a ventura version?

**Reply by SkyrilHD** [reply]
I will send you one in a bit.

**Reply by FireSpiritGM** [reply]
im gonna try the efi real quick

**Reply by FireSpiritGM** [reply]
oh ok

**Reply by FireSpiritGM** [reply]
the efi didnt work, the same issue, it shows me 2 options, an option named untitled and another option named MacOS (the name of the drive) then it black screens when i try to select anything or when i leave it for 10 seconds, then after maybe 10 mins it reboots

**Reply by FireSpiritGM** [reply]
hello?

**Reply by SkyrilHD** [reply]
Sorry, was busy.

Here you go: 
[Dell-Exx50-Hackintosh-3.2.Ventura-BumblebeeGPUDisable-NoAirportItlwm.zip](https://github.com/SkyrilHD/Dell-Exx50-Hackintosh/files/11653988/Dell-Exx50-Hackintosh-3.2.Ventura-BumblebeeGPUDisable-NoAirportItlwm.zip)

**Reply by FireSpiritGM** [reply]
ok thank you, im gonna test it now

**Reply by FireSpiritGM** [reply]
when i used macrecovery and followed the guide the macrecovery and typed cmd in the directory when cmd opened it didnt direct to the directory and it stayed in C:\  so i made it download the .dmg and chuncklist files onto another directory then copied the files into the root of the usb, can this be causing the issue? also the usb didnt show as a bootable device when i tried to boot, it just shows opencore and the blackout problem i mentioned before happens

**Reply by SkyrilHD** [reply]
I enabled verbose mode, check where it hangs: 
[Dell-Exx50-Hackintosh-3.2.Ventura-BumblebeeGPUDisable-NoAirportItlwm 2.zip](https://github.com/SkyrilHD/Dell-Exx50-Hackintosh/files/11654640/Dell-Exx50-Hackintosh-3.2.Ventura-BumblebeeGPUDisable-NoAirportItlwm.2.zip)

Also, maybe the files are corrupted, try redownloading them again.

**Reply by FireSpiritGM** [reply]
hello, i redownloaded the files, and tried to boot but i get a black screen after making a selection or after just waiting 4 seconds on the drive selection screen

**Reply by SkyrilHD** [reply]
You should get a screen like this after selecting the installer: 
![](https://dortania.github.io/OpenCore-Install-Guide/assets/img/1-boot-efi.853f625b.png)

If not, update your BIOS to the latest version and double-check the BIOS settings. https://github.com/SkyrilHD/Dell-Exx50-Hackintosh#recommended-bios-settings

**Reply by FireSpiritGM** [reply]
i bypassed microsoft's CPU requirements and TPM requirements and instaleld windows 11, so there isnt a bios for windows 11, what should i do, sorry for being a complete idiot

**Reply by FireSpiritGM** [reply]
i have the A24 bios which is latest i think

**Reply by SkyrilHD** [reply]
Enter the BIOS by pressing F10 during start up.

**Reply by FireSpiritGM** [reply]
is using macrecovery.py from opencore 0.9.2 while using ur efi on opencore 0.9 a potential cause of the issue?

**Reply by FireSpiritGM** [reply]
and i have made a floppy upgrade to a lenovo screen ( don't ask too much about it its kinda wonky ) can that be causing the issue?

**Reply by SkyrilHD** [reply]
I only provide support for stock configs. I cannot help you with any changes you made to the laptop, since I never had and will never have any experiences with it.

As a last solution from my side, try enabling 'Enable Legacy Option ROMs' in the BIOS: https://github.com/SkyrilHD/Dell-Exx50-Hackintosh#graphical-glitches-in-macos

**Reply by FireSpiritGM** [reply]
ok

**Reply by FireSpiritGM** [reply]
thanks for the suppport, ill take it over from here and try my best

**Reply by FireSpiritGM** [reply]
ive tried alot of different trouble shooting methods but to no avail, as a last resort, can you please send me a mediafire link containing the com.apple.recovery.boot folder with all of the needed files in it and the EFI which has the proper GPU disable?, im sorry to ask alot but i cant realyl seem to redownload the .dmg and chuncklist files

**Reply by FireSpiritGM** [reply]
hello, i was searching for a fix for the issue im having and i found someone that had pretty much the same issue, their explanation : After creating the boot thumbdrive with OpenCore.

To Reproduce
I created the bootdrive using Ubuntu and followed the [first method](https://dortania.github.io/OpenCore-Install-Guide/installer-guide/linux-install.html#method-1). Afterwards, I copied the BaseSystem.dmg and the BaseSystem.chunklist (which I got via macrecovery.py for Big Sur; tried Catalina too, same result) to the com.apple.recovery.boot folder at the root of the drive. After that I downloaded the EFI folder from this repo and placed it along the com.apple.recovery.boot folder at the root.

Then I restarted the laptop, went into BIOS and changed all the options from the README.

When choosing the thumbdrive as a startup device, I was greeted with OpenCore screen with icons, where I clicked the first one. Then the screen went black (but lit up). Sometimes it stays this way for more than 20 minutes until I manually turn off the laptop, other times it just reboots randomly. I tried using DEBUG version of OpenCore in hopes of getting the logs and finding the problem but to no avail (I replaced the files from the EFI folder from this repo to the files from the DEBUG release, as stated [here](https://dortania.github.io/OpenCore-Install-Guide/troubleshooting/debug.html#file-swaps): everything stays the same, even if I delete OpenCanopy.efi and remove the lines for it from config.plist.

Expected behavior
The installation starts.

Hardware:
ThinkPad T450, i5-5300U, HD Graphics 5500, 8 GB, Intel 7265,



ive tried alot of methods, i tried 3 efi folders, used the recommended bios settings and made 3 flash drives with the needed files but each one of them gives the same error but now its kinds of worse, because when i load into the opencore picker it only shows me the EFI partition on the usb and no hard drives, if u would like an ACPI path for the dGPU proper disable i can give it to you, it might be the fix

**Reply by FireSpiritGM** [reply]
this is the ACPI path i think

**Reply by FireSpiritGM** [reply]
`_SB_.PCI0.RP05.PEGP

**Reply by FireSpiritGM** [reply]
can anyone please reply with any helping tips?

**Reply by SkyrilHD** [reply]
You could try disabling GPU acceleration to check if the ig-platform-id is causing issues. Add this to boot-args: -igfxvesa

You could also try forcing the display to be online by adding this boot-arg: igfxonln=1

**Comment by emrah7celik** [comment]
I can confirm that this works on E5450. I also have activated WiFi and did install using WiFi. Thank you for sharing!

**Reply by Ginhuey** [reply]
The onboard wifi is working? Or did you need to change the card?

**Reply by SkyrilHD** [reply]
Glad to hear that. Will update the compatibility list soon.

**Reply by gaslarfa** [reply]
well, in the latest EFI update if it works natively on DELL E7450

**Reply by emrah7celik** [reply]
> The onboard wifi is working? Or did you need to change the card?

Intel WiFi is working, along with Bluetooth. I haven't made any change except for enabling itlwm in EFI config.

**Comment by fishudroid** [comment]
I can confirm that this work on E5450. But when i try dualboot with windows, the display was dim at every time boot in windows. Can you fix that problem and update for install the macOS beta 12 ?

**Reply by SkyrilHD** [reply]
How can I fix an issue that happens in Windows even though this EFI is aimed for macOS?

Everything you need to know about macOS Beta 12 is here: https://github.com/SkyrilHD/Dell-E7250-Hackintosh/discussions/27

**Reply by paddy-pyker** [reply]
The dim display has to do with opencore configuration being injected into windows. You can verify this by going to system information on windows and you'd realized the name of your pc model is MacBook pro.

For me, I got around this by booting windows with windows boot manager instead of opencore.

**Reply by SkyrilHD** [reply]
Ok thanks for comment, now I get the issue. I might fix it by making the ACPI Patches only working in macOS (no big promises though)

**Reply by SkyrilHD** [reply]
[SSDT-PNLF.aml.zip](https://github.com/SkyrilHD/Dell-Exx50-Hackintosh/files/6887832/SSDT-PNLF.aml.zip)

Can someone try replacing the old SSDT-PNLF file in EFI/OC/ACPI and see if the issue still persists? I don't have Windows on my machine, so I can't test it

**Reply by fishudroid** [reply]
oh okay i will test it

**Reply by fishudroid** [reply]
it doesn't work, it stills dim i don't know about this issue and i doesn't have mac to test again so at least tomorrow i'll install mac again to test.

**Reply by paddy-pyker** [reply]
[here](https://dortania.github.io/OpenCore-Install-Guide/why-oc.html#common-myths) 
Scrolling down on that page to 
**Does OpenCore always inject SMBIOS and ACPI data into other OSes?**

Reveals that you can disable the default behavior by setting the 
**CustomSMBIOSMode** to **Custom** 
under the **CustomSMIOSGuid** quirk in the config.plist 

Haven't tried that tho🤧

**Reply by SkyrilHD** [reply]
@ducltv can you try the possible solution mentioned above? Also, can you try disabling all ACPI injections and patches and check if Windows still boots to a dimmed screen?

**Reply by fishudroid** [reply]
@SkyrilHD hmm i don't have try that maybe tomorrow i install windows and try that because now i'm in macOS so i'll feedback soon as what i can.

**Comment by 2wenty2wo** [comment]
Confirming this works on **E5550**. Only issue I have is wifi dropping out after wake and not being able to reconnect without restarting. I'm not sure why it's doing this?

Edit: Yet randomly just then, it wakes from sleep and wifi is fine. Strange.

**Reply by quynkk5** [reply]
try to use Airportitlwm v1.2

**Reply by 2wenty2wo** [reply]
I was already using Airportitlwm, I updated to v1.3.0 stable with the same results.
I'm currently testing itlwm v1.3.0 stable with Heliport and it seems to be more stable? Kind of weird having two "wifi" symbols in the menubar, but it currently keeps connection after sleep. I'll continue testing.

**Reply by Bill-Joe-Tech** [reply]
> Confirming this works on **E5550**. Only issue I have is wifi dropping out after wake and not being able to reconnect without restarting. I'm not sure why it's doing this?
> 
> Edit: Yet randomly just then, it wakes from sleep and wifi is fine. Strange.

I tried on the E5550 and I am having problems. How were you able to get yours installed? For me I did the following and was unsuccessful: 

1. Installed Monterey via App Store.
2. Copied to the hard drive via the Apple command `sudo /Applications/Install\ macOS\ Monterey.app/Contents/Resources/createinstallmedia --volume /Volumes/MyVolume`
3. Made the EFI partition with MountEFI-update (as suggested in the [guide](https://dortania.github.io/OpenCore-Install-Guide/))
4. Copied the EFI folder to the EFI partition
5. Enabled AHCI
6. Booted the Hard drive
7. Formatted my hard drive as APFS
8. Installed macOS

With the last step, it would get to 12-13 minutes left then randomly reboot. Did I skip a step?

**Reply by Bill-Joe-Tech** [reply]
Figured out it was the Network Card I was using that was preventing the install. Switched it out with the OEM WiFi Card and was able to successfully install Ventura. 

Speeds happen to be somewhat decent for the OS being ran. Was able to install Xcode and ran a Hello World project with the Simulator.

**Comment by Sgrok17** [comment]
My E7450 works perfect on big sur, thanks, but how My WWAN can works too in hackintosh? i using HP lt4112 Gobi 4G Module.
![Annotation 2021-08-14 214316](https://user-images.githubusercontent.com/88937550/129450074-3c4b34fa-4901-43cd-b24e-eb017da6cf0e.png)

**Reply by SkyrilHD** [reply]
Try this: https://osxlatitude.com/forums/topic/8019-huawei-me906e-hp-lt4112-4g-modem-works-under-os-x/

**Reply by Sgrok17** [reply]
Yes. I was try this kext. Wwan detect as usb. But how to connect the internet?  Thanks

**Reply by Sgrok17** [reply]
No modem menu in network setting

**Reply by unclemate** [reply]
I met the same issue with lt4112 on BigSur. Did you fix it?

**Comment by saf111r** [comment]
I am sorry to say this. it's don't work on dell 7350 2 in 1 book. it’s stoped remaining 15 min When i install it. The installer logs show this: “verifying sharedsupport.dmg”.

**Reply by SkyrilHD** [reply]
1. Did you change the SMBIOS to MacBook8,1?
2. How did you create the installer?

**Reply by saf111r** [reply]
thanks to reply.
1. I changed the SMBIOS to MACBook8,1 with opencore configure tool.
2. i created the installer with balenaEtcher  and download the os dmg image from the internet.

![image](https://user-images.githubusercontent.com/26398705/132787175-a49e38e1-197e-4224-a454-b52f146d7136.png)

**Reply by saf111r** [reply]
by the way. I updated the BIOS version 1 years ago. Now the version is A17.

**Reply by SkyrilHD** [reply]
Download the installer from the App Store and create it with createinstallmedia. Something is wrong with your current installer. 

A17 is from 2018; there is A20 available.

**Reply by saf111r** [reply]
Thanks:
I successfully installed it.But the version is 11.5.2 (the only  version in App Store).
Still somethings don't work:

1. Bluetouch
2. no sounds (there is no output device )
3. touchscreen (added the kext)
4. The most strange things is that: After every first turn-on, the system seems very stutter and the USBS did not work. But is's will be fine after awaking the system again(waitting the system sleep about 10 min ). And the usbs also  work well.

I think is's the matter with USBMap.kext

Still thanks very much!

**Reply by SkyrilHD** [reply]
1. Did you enable IntelBluetooth Firmware?
2. Can you tell me which Codec you have. I guess it is different than on other models.
3. I included Touchscreen kexts, you just had to enable it. (Anything that said VoodooI2C in its name)
4. The EFI by default has USBInjectAll enabled (paired with FakePCIID and FakePCIID XHCI mux) which causes to break on 11.3+ because Apple changed something to break. Best option would be to create an USBMap.kext by yourself e.g with USBToolBox. (You could contribute to this project if you could send the USBMap.kext for your system)

**Reply by saf111r** [reply]
1. I enabled InterBluetooch Firmware
2. The Audio module is ALC3263(The same problem found there:  https://www.tonymacx86.com/threads/dell-latitude-7350-no-audio-big-sur-11-1-oc.311896/ ). I tried to inject alc-id with 1,3,11,13..
3. I enabled these  kexts,but it did't work.
4. I disable the default USBInjectAll and Use USBMap.kext with your notes. I will try to create my own USBMap.kext.(IF it is useful I will upload to the project)

Thanks.

**Comment by tomten1** [comment]
I have just installed on E5450, only one issue so far. I have the HD5500 and 1920x1080 display.

On boot I have a garbled screen but graphics acceleration is working.
If I Sleep and un-sleep, the screen gets sorted :)
I had the same issue when using the recovery installation method, connected an external HDMI monitor and was able to drag the required windows over to that monitor and was able to continue the setup.

Not sure there is a fix for this?

**Reply by paddy-pyker** [reply]
enabling legacy boot options in bios as suggested by @SkyrilHD  fixed mine

**Reply by tomten1** [reply]
Ah I had read that but thought I also needed to actually boot with non UEFI.

I enabled it and it worked as suggested! Great :)

**Comment by catalinabigsur** [comment]
Hi, I have used your EFI about 2 weeks now. And it works perfectly on my E7450. But I have a issues. It's about the ALC. Sometimes I can't hear the sound until I call Siri, it will work.

**Reply by SkyrilHD** [reply]
I also have this rare issue where the sound just cuts off (mostly happens when watching YouTube but the issue stopped long time ago). I just go to System Preferences>Sound>Input. I leave this window open and watch my video till the end. After closing System Preferences, sound still remains working. But I have no idea why it does that.

**Reply by shelterx** [reply]
Same issue here, First I thought my headphones were bad... but then I noticed it only seem to occur with some video clips or sites.

**Reply by catalinabigsur** [reply]
> I also have this rare issue where the sound just cuts off (mostly happens when watching YouTube but the issue stopped long time ago). I just go to System Preferences>Sound>Input. I leave this window open and watch my video till the end. After closing System Preferences, sound still remains working. But I have no idea why it does that.

But it will harm the battery life.

**Reply by SkyrilHD** [reply]
Howcome? Just because i left it open for 5 minutes?

**Comment by catalinabigsur** [comment]
Hi, when I try to upgrade my E7450 to Monterey Beta 9, I have kernel panic. I try to reset NVRAM but not success, can you fix it in next version?

**Reply by SkyrilHD** [reply]
Any logs? I am on Beta 8. I don't have enough space to update to Beta 9 😅

**Comment by catalinabigsur** [comment]
![b78a75175fd5968bcfc4](https://user-images.githubusercontent.com/91937101/136706315-74daee5e-7684-41a5-a9f7-0895cb2aa7fd.jpg)
Here it is, the kernel panic. I'm not sure why...

**Reply by SkyrilHD** [reply]
Enable Verbose mode by adding '-v' (without quotes) to boot-args in config.plist

**Reply by catalinabigsur** [reply]
Ok, let me try, but it is a solution or you want to know what are the problem?

**Reply by SkyrilHD** [reply]
I want to know what the issue is. Currently, I don't know. Which EFI version are you using btw?

**Reply by catalinabigsur** [reply]
I am using the newest 0.7.4. Here it's the EFI that I modify on my E7450: https://drive.google.com/drive/folders/14ZjVmQN8z9l7k7RSl8T1Wor95wA1iS3B?usp=sharing
Hope you will fix it. Good luck!

**Reply by SkyrilHD** [reply]
But could you still post a verbose log, so I know what the issue is?

**Reply by catalinabigsur** [reply]
Sorry, but I have downgrade to Big Sur. When on Monterey it keeps restart.

**Reply by catalinabigsur** [reply]
Have you done right?

**Comment by catalinabigsur** [comment]
This is a kernel panic details on my E7450:
panic(cpu 1 caller 0xffffff801e1e5b93): Kernel trap at 0xffffff801e81b483, type 14=page fault, registers:
CR0: 0x0000000080010033, CR2: 0x0000000000000018, CR3: 0x00000000d85a602d, CR4: 0x00000000003626e0
RAX: 0xffffff8021f9d148, RBX: 0x0000000000000000, RCX: 0x0000000000000000, RDX: 0x0000000000420000
RSP: 0xffffffd07286bc60, RBP: 0xffffffd07286bc70, RSI: 0xffffff853dc8d6c0, RDI: 0x0000000000000000
R8:  0x0000000000000016, R9:  0x0000000000000002, R10: 0xffffff92b6858420, R11: 0xffffff801e9e4220
R12: 0xffffff801f06a6e8, R13: 0x00000000e00002bd, R14: 0xffffff8540d76580, R15: 0xffffff92b746a03c
RFL: 0x0000000000010246, RIP: 0xffffff801e81b483, CS:  0x0000000000000008, SS:  0x0000000000000010
Fault CR2: 0x0000000000000018, Error code: 0x0000000000000000, Fault CPU: 0x1, PL: 0, VF: 0
Panicked task 0xffffff853f506d40: 4 threads: pid 97: systemstats
Backtrace (CPU 1), panicked thread: 0xffffff8540d76580, Frame : Return Address
0xffffffd07286b690 : 0xffffff801e09c06d 
0xffffffd07286b6e0 : 0xffffff801e1f5dc5 
0xffffffd07286b720 : 0xffffff801e1e57a3 
0xffffffd07286b770 : 0xffffff801e03ba60 
0xffffffd07286b790 : 0xffffff801e09c43d 
0xffffffd07286b8b0 : 0xffffff801e09bbf6 
0xffffffd07286b910 : 0xffffff801e918589 
0xffffffd07286b980 : 0xffffff801e1e5b93 
0xffffffd07286bb00 : 0xffffff801e1e5878 
0xffffffd07286bb50 : 0xffffff801e03ba60 
0xffffffd07286bb70 : 0xffffff801e81b483 
0xffffffd07286bc70 : 0xffffff8020bc9e46 
0xffffffd07286bcb0 : 0xffffff801e820f94 
0xffffffd07286bcd0 : 0xffffff801e8214af 
0xffffffd07286bd00 : 0xffffff801e895b48 
0xffffffd07286bd70 : 0xffffff801e1a1b66 
0xffffffd07286bdc0 : 0xffffff801e076e19 
0xffffffd07286be50 : 0xffffff801e08dde9 
0xffffffd07286bee0 : 0xffffff801e1cab32 
0xffffffd07286bfa0 : 0xffffff801e03c246 
      Kernel Extensions in backtrace:
         com.apple.iokit.IOPCIFamily(2.9)[10E029BC-D573-361D-A867-DACDD4120FD0]@0xffffff8020bbc000->0xffffff8020be6fff
Process name corresponding to current thread (0xffffff8540d76580): systemstats
Boot args: -wegnoegpu chunklist-security-epoch=0 -chunklist-no-rev2-dev
Mac OS version:
21A5543b
Kernel version:
Darwin Kernel Version 21.1.0: Thu Sep 30 06:41:59 PDT 2021; root:xnu-8019.40.96.0.2~12/RELEASE_X86_64
Kernel UUID: 01C23ADA-6926-3178-A9B6-F0FA57FD14A4
KernelCache slide: 0x000000001de00000
KernelCache base:  0xffffff801e000000
Kernel slide:      0x000000001de10000
Kernel text base:  0xffffff801e010000
__HIB  text base: 0xffffff801df00000
System model name: MacBookPro12,1 (Mac-E43C1C25D4880AD6)
System shutdown begun: NO
Panic diags file available: YES (0x0)
Hibernation exit count: 0
System uptime in nanoseconds: 469925361349
Last Sleep:           absolute           base_tsc          base_nano
  Uptime  : 0x0000006d69bbfbf4
  Sleep   : 0x0000000000000000 0x0000000000000000 0x0000000000000000
  Wake    : 0x0000000000000000 0x0000000a61307282 0x0000000000000000
Zone info:
Foreign   : 0xffffff802a360000 - 0xffffff802a36d000
Native    : 0xffffff803048f000 - 0xffffffa03048f000
Readonly  : 0 - 0
Metadata  : 0xffffffd227cbf000 - 0xffffffd247d21000
Bitmaps   : 0xffffffd247d21000 - 0xffffffd249521000


Hope you will fix it. Good luck!
Although I use -v, there is no diffrence.

**Comment by Dan1704** [comment]
I can confirm this EFI config works on the E5250 without any modification. However only 1 issue I have come across is the wifi not being recognised. This aside, everything else works a treat!

**Reply by SkyrilHD** [reply]
Did you enable AirportItlwm inside config.plist?

**Comment by KCVEMURI** [comment]
I confirm that EFI works on E5450 , however i see below text during boot though it gets booted . I am using dual boot with windows 10 and MAC Monterey. Please help me removing these text during boot

OCS: No schema for AllowNvramReset at 0 index, context <security>!
OCS: No schema for AllowtoggleSip at 2 index, context <security>!

**Reply by SkyrilHD** [reply]
Typo. It is ResetNvramEntry.efi and ToggleSipEntry.efi and you need to remove AllowToggleSip and AllowNvramReset.

Reference: https://github.com/SkyrilHD/Dell-Exx50-Hackintosh/compare/5131cf01c9382eb328541a08826e57f4a8dfef11...b3a254530ca57002fb84616b04e87aaa9550bf8d

**Reply by KCVEMURI** [reply]
Thanks for the reply. Do i need to set them as true as shown in Reference: https://github.com/SkyrilHD/Dell-Exx50-Hackintosh/compare/5131cf01c9382eb328541a08826e57f4a8dfef11...b3a254530ca57002fb84616b04e87aaa9550bf8d? sorry for asking a naive question but i am very new to this!!

<key>AllowToggleSip</key>
			<true/>

right now in my plist it is set as false

**Reply by SkyrilHD** [reply]
You don't have to, but I did it for consistency. Removing AllowToggleSip and AllowNvramReset is enough.

But if you want to use these tools, you need to paste ResetNvramEntry.efi and ToggleSipEntry.efi to the drivers folder and then change the config shown in the reference.

**Reply by KCVEMURI** [reply]
i tried setting these entries to true yet i get the same message on boot. i also have ResetNvramEntry.efi and ToggleSipEntry.efi in the drivers folder. please let me know what else i am missing

**Reply by SkyrilHD** [reply]
Follow the reference commits, then the errors will be gone.

**Reply by KCVEMURI** [reply]
After removing those lines from plist the error is gone and my E5450 is running smooth. thanks for all the help.

**Comment by ghost** [comment]
> This discussion about the Exx50 models. Since I only own an E7250 which is the only device I can test on, I do not know if the EFI is compatible with other models of Exx50 models. So, it would be useful if you could answer with the model you have and whether it working or not. If the EFI is not working, consider reporting the issue which I would look into.

The EFI has complete compatibility with the e7450. For those using the EFI, go and do a clean snapshot after enabling airportwm kext. Once rebooted, all hardware is enabled. I decided to take the legacy hardware support to fix the screen issues, works wonderfully!

**Reply by Swearie** [reply]
What do you mean by "do a clean snapshot"? Reinstalling the whole system?

**Comment by AbdulrahmanAli19** [comment]
![Screenshot 2022-11-02 at 3 37 14 PM](https://user-images.githubusercontent.com/60023467/199503995-38496a2d-fc7a-4cd4-bd07-5bee3bbc80cd.png)
Its fully working on latitude e5450, I have updated from Monetary to Ventura easily thanks for your amazing support of this repository

**Comment by Swearie** [comment]
I have a e7450 and Ventura installed. Everything works except Wi-Fi and Bluetooth. I've already tried everything suggested. Also, everytime I reboot the system, it shows warnings like I suddenly turned it down instead of doing it the right way. The thing I'd need the most is WiFi of course

Edit: nevermind, everything's working, I was just doing something wrong in the config.plist. Thanks a lot for the amazing work!

**Reply by Skye10da** [reply]
Hi, I'm using e7450 too but I'm on big sur, how's your performance experience

**Comment by robrobpenguin** [comment]
I own an e5250 with Monterey installed. The WIFI and Bluetooth didn't work at first until I enabled them in the config.plist. Thank you for making my life easier!

**Comment by Atonify** [comment]
> This discussion about the Exx50 models. Since I only own an E7250 which is the only device I can test on, I do not know if the EFI is compatible with other models of Exx50 models. So, it would be useful if you could answer with the model you have and whether it working or not. If the EFI is not working, consider reporting the issue which I would look into.

well it kind of works on Latitude E7450 but the screen is like weird there are multiples and its like half of the pixels on the image are gone idk if its my laptop of smth else but it worked fine with windows

**Reply by SkyrilHD** [reply]
I think the issue you describing is this: https://github.com/SkyrilHD/Dell-Exx50-Hackintosh#graphical-glitches-in-macos

**Comment by longh1003** [comment]
> This discussion about the Exx50 models. Since I only own an E7250 which is the only device I can test on, I do not know if the EFI is compatible with other models of Exx50 models. So, it would be useful if you could answer with the model you have and whether it working or not. If the EFI is not working, consider reporting the issue which I would look into.
Thanks man,  it works well. But it works only by Ethernet. Wifi is unavailable(neither AirportItlwm or Itlwm). Could you help me check? Peace!! =)
Here is my specs
Latitude e5450
macOS Monterey Version 12.6.3 (Build 21G419)
Intel(R) Core(TM) i3-5010U CPU @ 2.10GHz 
Broadwell
Intel HD Graphics 5500
Qualcomm Atheros
VSP-240G QVE

**Reply by fishudroid** [reply]
Airportitlwm or even Itlwm just worked with intel wifi card. Ur system has Qualcomm Atheros which mean it doesn't support. You should change the wifi card to a BCM943xxxx or an intel wifi card and use AirportItlwm or Itlwm

**Reply by longh1003** [reply]
Thanks for your reply! Appreciate that!

**Comment by alexwajn** [comment]
Trying to install it on my E5550. (5300U w/ Intel graphics).

I made the USB stick using the Windows process, pasted the contents of this EFI into the root of the drive, modified BIOS. The picker was not showing "Install MacOS" but whenever I press space bar, it would show an icon with a gear and the volume label of the drive and dmg. Whenever I start this icon, the apple logo appears, after a while a progress bar shows up, advances a bit but freezes. 

Tried both on Sequoia and Monterey

Am I missing some step? I much appreciate any help on this.

**Comment by nabilh2020** [comment]
On Dell Latitude E5550 it works with minor tinkering.
In Ventura everything works.
In Sequoia WiFi works only with Itlwm & Heliport app.
My comments posted here:
https://github.com/SkyrilHD/Dell-Exx50-Hackintosh/discussions/96
